src/query-builder/SelectQueryBuilder.ts

```diff
@@ -1929,6 +1937,16 @@ export class SelectQueryBuilder<Entity> extends QueryBuilder<Entity> implements
                 }
             }
 
+            if (this.findOptions.lock) {
```

Comment:
@Kononnable did you have this change already? I need to know if I should move this change or not.

Reply:
Yes, it's there.
https://github.com/typeorm/typeorm/blob/ba4bebe5d9045ec8655ca50e0e022fb606149f13/src/query-builder/SelectQueryBuilder.ts#L1985-L1991